specs/paymentrequest.html

```diff
@@ -284,16 +284,77 @@
           payment app.
         </div>
         <div class="note">
-          <p>The <code>supportedMethods</code> sequence contains the <a>payment method identifiers</a>
-          for the <a>payment methods</a> that the merchant web site accepts.</p>
+
+          <p>The <code>details</code> contains the core payment request data that will be passed on to the user selected
+            <a>payment app</a>.</p>
+
+          <p>The <code>amount</code> sequence contains the amount (possibly in different currencies) that the website
```

Comment:
I'm inclined to leave it as a top element (initially I think it will be seldom overridden) and make the per method member optional (which I think makes it clearer that it's an override).